java/client/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -236,4 +241,17 @@ public void launchApp(String id) {
     execute(ChromeDriverCommand.LAUNCH_APP, ImmutableMap.of("id", id));
   }
 
+  /**
+   * Send command for browser to allow downloading in headless mode
+   *
+   * @param downloadPath
+   */
+  public void sendCommandForDownloadChromeHeadLess(String downloadPath) {
+    if (downloadPath.length()>0){
```

Comment:
What happens if the path is zero length? That seems like an unusual position to be in. Log or throw an error?